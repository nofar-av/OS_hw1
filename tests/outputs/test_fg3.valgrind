==27014== Memcheck, a memory error detector
==27014== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==27014== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==27014== Command: /home/omer/OS_wet1/smash
==27014== Parent PID: 27008
==27014== 
==27014== 
==27014== FILE DESCRIPTORS: 4 open at exit.
==27014== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_fg3.valgrind
==27014==    <inherited from parent>
==27014== 
==27014== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_fg3.err
==27014==    <inherited from parent>
==27014== 
==27014== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_fg3.out
==27014==    <inherited from parent>
==27014== 
==27014== Open file descriptor 0:
==27014==    <inherited from parent>
==27014== 
==27014== 
==27014== HEAP SUMMARY:
==27014==     in use at exit: 0 bytes in 0 blocks
==27014==   total heap usage: 91 allocs, 91 frees, 85,608 bytes allocated
==27014== 
==27014== All heap blocks were freed -- no leaks are possible
==27014== 
==27014== For counts of detected and suppressed errors, rerun with: -v
==27014== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
