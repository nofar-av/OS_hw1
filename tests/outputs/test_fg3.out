smash> smash> smash> smash> smash> smash> sleep 300& : 5480
smash: got ctrl-C
smash: process 5480 was killed
smash> [1] sleep 100& : 5477 0 secs
[2] sleep 200& : 5479 0 secs
[4] sleep 400& : 5481 0 secs
[5] sleep 500& : 5482 0 secs
smash> sleep 100& : 5477
smash: got ctrl-C
smash: process 5477 was killed
smash> [2] sleep 200& : 5479 1 secs
[4] sleep 400& : 5481 1 secs
[5] sleep 500& : 5482 1 secs
smash> sleep 200& : 5479
smash: got ctrl-C
smash: process 5479 was killed
smash> [4] sleep 400& : 5481 1 secs
[5] sleep 500& : 5482 1 secs
smash> sleep 500& : 5482
smash: got ctrl-C
smash: process 5482 was killed
smash> [4] sleep 400& : 5481 2 secs
smash> sleep 400& : 5481
smash: got ctrl-C
smash: process 5481 was killed
smash> smash> smash: sending SIGKILL signal to 0 jobs:
